src/extension/mapml/src/test/java/org/geoserver/mapml/MapMLTestSupport.java

```diff
@@ -119,6 +122,18 @@ public boolean isFeature() {
             return feature;
         }
 
+        public boolean isFeatureRep() {
```

Comment:
"rep" again, if there is any renaming in the constants, I'd match it here.

Reply:
Changed to new (hopefully clearer) terminology.